test/objects.test.js

```diff
@@ -0,0 +1,169 @@
+import { InfoBox, Link } from "../js/objects";
+
+describe("InfoBox", () => {
+  it("should initialize with correct values", () => {
+    const id = 1;
+
+    const infoBox = new InfoBox(id);
+
+    expect(infoBox.id).toBe(id);
+    expect(infoBox.x).toBe(0);
+    expect(infoBox.y).toBe(0);
+    expect(infoBox.width).toBe(120);
+    expect(infoBox.height).toBe(240);
+    expect(infoBox.lineColor).toBe("black");
+    expect(infoBox.lineWidth).toBe(2);
+    expect(infoBox.color).toBe("white");
+    expect(infoBox.row).toBe(-1);
+    expect(infoBox.texImg).toBe(null);
+    expect(infoBox.name).toBe("");
+    expect(infoBox.momentum).toBe(0);
+    expect(infoBox.px).toBe(0);
+    expect(infoBox.py).toBe(0);
+    expect(infoBox.pz).toBe(0);
+    expect(infoBox.vertex).toBe(0);
+    expect(infoBox.vx).toBe(0);
+    expect(infoBox.vy).toBe(0);
+    expect(infoBox.vz).toBe(0);
+    expect(infoBox.time).toBe(0);
+    expect(infoBox.mass).toBe(0);
+    expect(infoBox.charge).toBe(0);
+    expect(infoBox.pdg).toBe(0);
+    expect(infoBox.genStatus).toBe(0);
+    expect(infoBox.simStatus).toBe(0);
+    expect(infoBox.parents).toEqual([]);
+    expect(infoBox.children).toEqual([]);
+    expect(infoBox.parentLinks).toEqual([]);
+    expect(infoBox.childrenLinks).toEqual([]);
+  });
+
+  it("should return true if (x, y) coordinates are within the box", () => {
+    const infoBox = new InfoBox(1);
+    const x = 60;
+    const y = 120;
+
+    const result = infoBox.isHere(x, y);
+
+    expect(result).toBe(true);
+  });
+
+  it("should return false if x coordinate is outside the box", () => {
+    const infoBox = new InfoBox(1);
+    const x = 200;
+    const y = 120;
+
+    const result = infoBox.isHere(x, y);
+
+    expect(result).toBe(false);
```

Comment:
```suggestion
    expect(infoBox.isHere(x, y)).toBe(false);
```
Would make the cases a few lines shorter. Similar for other cases below.